### ISSUE TITLE
Allow PHPUnit 8

### ISSUE THREAD

**Issue by MichielDilissen** [matthiasnoback/doctrine-dbal-test-service-provider#2]
Is it possible to support PHPUnit 8 (maybe changing the requirements could suffice)?

**Comment by matthiasnoback** [comment]
Feel free to submit a PR. We can just release a new minor version for that if needed.

**Comment by MichielDilissen** [comment]
I encountered a dependency issue while trying to increase the version of PHPUnit. In order to do this another of your packages needs to be updated. Because this has to be merged first, I also made a PR to do this: https://github.com/matthiasnoback/phpunit-test-service-container/pull/6.

When that PR is merged I will make another PR to update this package.

**Comment by matthiasnoback** [comment]
Fixed in #3.
